src/Constants.ts

```diff
@@ -138,6 +139,7 @@ export const ClassNames = {
     RDB: 'RDB',
     REDIS: 'Redis',
     HTTP: 'HTTP',
+    HTTP2: 'HTTP2',
```

Comment:
I think, it is better to use `HTTP` class name without defining `HTTP2` class name